.tekton/pull-request.yaml

```diff
@@ -83,9 +83,43 @@ spec:
               workingDir: $(workspaces.source.path)
               script: |
                 #!/usr/bin/env bash
-                ./hack/check-partner-tasks.sh
+                ./hack/check-partner-tasks.sh > partner-tasks.out
+                echo $? > partner-tasks-code.out
+                cat partner-tasks.out
+            - name: create-comment
+              image: quay.io/chmouel/github-app-token@sha256:b4f2af12e9beea68055995ccdbdb86cfe1be97688c618117e5da2243dc1da18e
```

Comment:
Would it be good to move the image under `redhat-appstudio` org?

Reply:
moved